lib/tree_adapters/default.js

```diff
@@ -512,6 +512,23 @@ exports.getDocumentTypeNodeSystemId = function (doctypeNode) {
     return doctypeNode.systemId;
 };
 
+/**
+ * Returns the given template node's content.
+ *
+ * @function getTemplateContent
+ * @memberof TreeAdapter
+ *
+ * @param {ASTNode} node - Node.
+ *
+ * @returns {ASTNode}
+ *
+ * @see {@link https://github.com/inikulin/parse5/blob/tree-adapter-docs-rev/lib/tree_adapters/default.js#L527|default implementation.}
+ */
+exports.getTemplateContent = function (node) {
```

Comment:
No need for this, it's already implemeted: https://github.com/yyx990803/parse5/blob/master/lib/tree_adapters/default.js#L166